contrib/crane/template.py

```diff
@@ -0,0 +1,43 @@
+pkgname = "crane"
+pkgver = "0.20.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    "./cmd/crane",
+]
+hostmakedepends = [
+    "go",
+]
+pkgdesc = "Container image and registry manipulation tool"
+maintainer = "Radosław Piliszek <radek@piliszek.it>"
+license = "Apache-2.0"
+url = "https://github.com/google/go-containerregistry"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "4bae53f34011e35ef874a60123b8ae70a5e992d804decb030479dbb888afe6d1"
+
+
+_supported_shells = ["bash", "zsh", "fish"]
```

Comment:
drop and just put in the for loops

Reply:
why? it would not be DRY then

Reply:
because it increases verbosity and adds no value

Reply:
duplicated code increases verbosity more but I don't mind that much in this particular situtation

Reply:
it does not; it results in 3 lines being gone and people not having to look away for context, and when adding to it updating both places is impossible to miss, defeating the entire purpose of DRY